### PR TITLE
endpoint to read teams

### PR DESCRIPTION
**Pull request by Tatianeb**

### CODE REVIEW

File: src/copa_game/adapters.clj [hunk 1]
```diff
@@ -0,0 +1,7 @@
+(ns copa-game.adapters
+  (:require [schema.core :as s]))
+
+(s/defn wire->teams
+        [wire]
+  (def wire wire)
```
Comment (vloth) [inline src/copa_game/adapters.clj]: ops, we should remove this `def`.

**Review by vloth — CHANGES_REQUESTED** [review]
I think we could remove `.cpcache` and `.nrepl-port` from the source code and ignore them on `.gitignore` too. They change frequently and don't need to be in the source code.
I think it would be also nice to add `clj-kondo` and `clojure-lsp`.

Answear: Nice. Add clj-kondo and clojure-lsp in gitignore too?

File: src/copa_game/adapters.clj (hunk 1 above)
Comment (Tatianeb) [reply src/copa_game/adapters.clj]: ✂️

**Comment by vloth** [comment]
looks good to me!

**Pull request merged**